app.py

```diff
@@ -114,11 +114,9 @@ def run_solver(solver: str, packages: str) -> str:
                 "env": [
                     {"name": "THOTH_SOLVER", "value": str(solver)},
                     # No need to run transitive again
-                    {"name": "THOTH_SOLVER_NO_TRANSITIVE", "value": "1"},
-                    {"name": "THOTH_SOLVER_PACKAGES", "value": str(
-                        packages.replace('\n', '\\n'))},
-                    # TODO: hardcoded now
-                    {"name": "THOTH_SOLVER_DEBUG", "value": "1"},
+                    {"name": "THOTH_SOLVER_NO_TRANSITIVE", "value": "0"},
```

Comment:
why no transitives?

Reply:
This will turn on transitive solving.

Reply:
ja, why not?

Reply:
If a transitive dependency introduces a dependency that was not previously analyzed, we do not have full stack in the graph database.